<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEnumerationDialog</class>
 <widget class="QDialog" name="EditEnumerationDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>446</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/topcased_icons/Enumeration_24.gif</normaloff>:/topcased_icons/Enumeration_24.gif</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
     <property name="buddy">
      <cstring>nameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Values</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QPushButton" name="removePushButton">
        <property name="text">
         <string>&amp;Remove</string>
        </property>
        <property name="icon">
         <iconset theme="list-remove" resource="../resources/resources.qrc">
          <normaloff>:/kde_icons/list-remove.png</normaloff>:/kde_icons/list-remove.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="addPushButton">
        <property name="text">
         <string>&amp;Add</string>
        </property>
        <property name="icon">
         <iconset theme="list-add" resource="../resources/resources.qrc">
          <normaloff>:/kde_icons/list-add.png</normaloff>:/kde_icons/list-add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="valueLineEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="moveUpPushButton">
        <property name="text">
         <string>Move &amp;Up</string>
        </property>
        <property name="icon">
         <iconset theme="arrow-up" resource="../resources/resources.qrc">
          <normaloff>:/kde_icons/arrow-up.png</normaloff>:/kde_icons/arrow-up.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="moveDownPushButton">
        <property name="text">
         <string>Move &amp;Down</string>
        </property>
        <property name="icon">
         <iconset theme="arrow-down" resource="../resources/resources.qrc">
          <normaloff>:/kde_icons/arrow-down.png</normaloff>:/kde_icons/arrow-down.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" rowspan="5">
       <widget class="QListWidget" name="valuesListWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>valueLineEdit</tabstop>
  <tabstop>valuesListWidget</tabstop>
  <tabstop>addPushButton</tabstop>
  <tabstop>removePushButton</tabstop>
  <tabstop>moveUpPushButton</tabstop>
  <tabstop>moveDownPushButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditEnumerationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditEnumerationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPushButton</sender>
   <signal>clicked()</signal>
   <receiver>EditEnumerationDialog</receiver>
   <slot>slotAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removePushButton</sender>
   <signal>clicked()</signal>
   <receiver>EditEnumerationDialog</receiver>
   <slot>slotRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveUpPushButton</sender>
   <signal>clicked()</signal>
   <receiver>EditEnumerationDialog</receiver>
   <slot>slotMoveUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveDownPushButton</sender>
   <signal>clicked()</signal>
   <receiver>EditEnumerationDialog</receiver>
   <slot>slotMoveDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valuesListWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>EditEnumerationDialog</receiver>
   <slot>slotSelectedRowChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valueLineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>EditEnumerationDialog</receiver>
   <slot>slotEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotAdd()</slot>
  <slot>slotRemove()</slot>
  <slot>slotMoveUp()</slot>
  <slot>slotMoveDown()</slot>
  <slot>slotSelectedRowChanged()</slot>
  <slot>slotEdited()</slot>
 </slots>
</ui>
